PerfDataExtension/PerfDataFilesTable.cs

```diff
@@ -0,0 +1,236 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Performance.Toolkit.Plugins.PerfDataExtension

```

Comment:
Not related to this file. But please also modify:

azure-pipelines.yml
- Add both projects (main and Unit Test) to "DotNetCore build"
- Copy/create a CopyFiles to get a build output folder
         - task: CopyFiles@2
          displayName: Copy PerfDataExt Build to Output Artifacts